packages/material/scss/chat/_theme.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@progress/kendo-theme-default/scss/chat/theme";
+
+@include exports("chat/theme/theme") {
+
+    // Chat
+    .k-chat {
+
+
+        // Bubbles
+        .k-bubble {
+            line-height: 1.25;
```

Comment:
Seems like the `line-height: 1.25` is found both here and in bootstrap. Introducing a variable for these in the default theme can shave off these rules.